specification/consumption/resource-manager/Microsoft.Consumption/stable/2021-05-01/examples/LotsListByBillingAccountWithFilters.json

```diff
@@ -13,17 +13,32 @@
             "name": "lot1",
             "type": "Microsoft.Consumption/lots",
             "properties": {
+              "creditCurrency": "USD",
```

Comment:
Please revert all the changes on this file.

Reply:
Reverted.